ddtrace/settings/config.py

```diff
@@ -805,15 +809,92 @@ def _format_tags(self, tags: List[Union[str, Dict]]) -> Dict[str, str]:
             pairs = [t.split(":") for t in tags]  # type: ignore[union-attr,misc]
         return {k: v for k, v in pairs}
 
+    def _tracerFlarePubSub(self):
```

Comment:
These four functions don't actually have much to do with "config" as we use that term in ddtrace. Moving them to `internal/flare` and importing them here might reduce the cognitive load required to understand what this file is doing.

Reply:
Sounds good, I'll go ahead and move them over